stashpr.py

```diff
@@ -59,7 +61,7 @@ def __init__(self, project, repository, onto, branch, state,
         self.branch = branch
         self.state = state
         self.title = title or re.sub('^refs/heads/', '', branch)
-        self.description = description or find_description(onto, branch)
+        self.description = description or find_description('HEAD', onto)
```

Comment:
there's a cli argument to change the branch so using HEAD here is not always going to be the correct thing.